mobile-app/app/screens/AppNavigator/screens/Dex/components/DexScrollable.tsx

```diff
@@ -152,10 +152,10 @@ function TotalLiquidityValue({
         </ThemedTextV2>
       ) : (
         <NumberFormat
-          value={value}
-          decimalScale={0}
+          value={BigNumber(value).toFixed(2)}
           thousandSeparator
           displayType="text"
+          valueIsNumericString
```

Comment:
does it work if we remove this?

Reply:
if removed valueIsNumericString it works as well to 2dp
<img width="191" alt="image" src="https://user-images.githubusercontent.com/57183851/197913326-35061d16-d8e1-4185-9881-285d26561d7b.png">

i also found out that when we use toFixed(2) it would append .00 to whole values. Would that be fine too 
<img width="374" alt="image" src="https://user-images.githubusercontent.com/57183851/197913450-26a2331f-385f-4e0b-aec2-a774632a37a0.png">